.github/workflows/dependabot.yml

```diff
@@ -1,5 +1,10 @@
 name: Merge dependabot PRs automatically
-on: pull_request_target
+on:
+  pull_request_target:
+    paths:
+      - '!rbs.gemspec'
+      - '!Gemfile'
+      - '!steep/Gemfile'
```

Comment:
Here: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore